hdfs/client.py

```diff
@@ -293,6 +294,21 @@ def acl_status(self, hdfs_path, strict=True):
     res = self._get_acl_status(hdfs_path, strict=strict)
     return res.json()['AclStatus'] if res else None
 
+  def set_acl(self, hdfs_path, aclspec):
+    """Set ACL for specified path. Returns None for successful executions.
```

Comment:
To be consistent with the other `set_???` methods, could we make this method not return anything?